docs/topics/selectors.rst

```diff
@@ -808,8 +821,8 @@ nodes can be accessed directly by their names:
 
     >>> response.selector.remove_namespaces()
     >>> response.xpath("//link")
-    [<Selector xpath='//link' data='<link rel="alternate" type="text/html" h'>,
-        <Selector xpath='//link' data='<link rel="next" type="application/atom+'>,
+    [<Selector query='//link' data='<link rel="alternate" type="text/html" h'>,
+        <Selector query='//link' data='<link rel="next" type="application/atom+'>,
```

Comment:
This falls within  a skip section, so it would not have been caught. I did not check why, though, so maybe it needs to be skipped?

Reply:
This section is skipped because the code deals with an external source (we could include a sample XML file with namespaces into the doc instead, but that's out of scope here I think), but I don't see why we shouldn't change the expected output :) A user who runs this with recent Parsel will see `query=`.